### ISSUE TITLE
block-closing-brace-newline-after throw warning on Sass `if/else` construction

### ISSUE THREAD

**Issue by fliptheweb** [stylelint/stylelint#798]
If we're have `block-closing-brace-newline-after: always` in rules and Sass `if/else` construction, e.g:
```sass
@mixin mixin($arg: false) {
  @if $arg {
    
  } @else {
    
  }
}
```
`stylelint 4.4.0` throw warning about that:
```sh
Expected newline after "}" (block-closing-brace-newline-after)
```

Perhaps `stylelint` could see the difference between [sass control directives & expressions](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#control_directives__expressions) like `@if` and CSS blocks?

**Comment by davidtheclark** [comment]
Duplicate of #561

**Comment by jeddy3** [comment]
As @davidtheclark said, this is also a duplicate. Adding the necessary options proposed in #561 would be a great way to contribute. It is a relatively simple change and there are plenty of examples of similar options in other rules.
